edgegrid/log.go

```diff
@@ -85,24 +98,53 @@ func LogMultilinef(f func(formatter string, args ...interface{}), formatter stri
 // Utility func to print http req
 func PrintHttpRequest(req *http.Request, body bool) {
 
-        if req == nil {
-                return
-        }
-        b, err := httputil.DumpRequestOut(req, body)
-        if err == nil {
-                LogMultiline(EdgegridLog.Traceln, string(b))
-        }
+	if req == nil {
+		return
+	}
+	b, err := httputil.DumpRequestOut(req, body)
+	if err == nil {
+		LogMultiline(EdgegridLog.Traceln, string(b))
```

Comment:
Isn't this logging the same request two different ways?  One as a single line out and another per line of the output?  I like logging the request as a single action but am unable to grok why we still need it the old way as well.  Can you explain?

Reply:
Ok I can remove these that was initial testing of wrapped logs I broke it out to separate function to allow incremental migration once correlation concept catches on